composer.json

```diff
@@ -9,8 +9,7 @@
         "react/event-loop": "^0.4",
         "react/stream": "^0.4",
         "react/http": "dev-master#cd15204bd15d106d7832c680e4fb0ca0ce2f5e30",
-        "react/socket-client": "^0.5.0",
-        "react/socket": "^0.5.0",
+        "react/socket": "^0.8|^1.0",
```

Comment:
1.0 isn't out yet, so I wouldn't add that yet.

Reply:
It's actually what ratchet does and I wanted to stay compatible with that. But it's a risk, I can remove it.

Reply:
The ReactPHP does that at several places for whatever reason.